repo/blob/sftp/sftp_storage.go

```diff
@@ -202,15 +201,22 @@ func (s *sftpImpl) PutBlobInPath(ctx context.Context, dirPath, fullPath string,
 			return errors.Wrap(err, "unexpected error renaming file on SFTP")
 		}
 
-		return nil
-	})
-}
+		if t := opts.SetModTime; !t.IsZero() {
+			if chtimesErr := sftpClientFromConnection(conn).Chtimes(fullPath, t, t); err != nil {
+				return errors.Wrap(chtimesErr, "can't change file times")
```

Comment:
```suggestion
				return errors.Wrapf(chtimesErr, "can't change file %q times", fullPath)
```

Reply:
same suggestion on other fs-op failures, having the error and also the full path is helpful on returned errors

Reply:
i think all these errors already mention the path.